patch/generate/llvmIntrinsicEmu.ll

```diff
@@ -29,6 +29,15 @@ define float @tanf(float %angle) #0
     ret float %1
 }
 
+; GLSL: float ldexpf(float, int)
+define float @ldexpf(float %x, i32 %exp)
+{
+    ; This function is generated by LLVM call simplification when handling pow(2.0, exp).
```

Comment:
please remove when handling pow(2.0, exp). in comments, it doesn't match your case